scripts/west_commands/runners/canopen_program.py

```diff
@@ -63,14 +64,18 @@ def name(cls):
 
     @classmethod
     def capabilities(cls):
-        return RunnerCaps(commands={'flash'}, flash_addr=False)
+        return RunnerCaps(commands={'flash'}, dev_id=True, flash_addr=False)
 
     @classmethod
-    def do_add_parser(cls, parser):
-        # Required:
-        parser.add_argument('--node-id', required=True, help='Node ID')
+    def dev_id_help(cls) -> str:
+        return 'CAN Node ID.'
```

Comment:
```suggestion
        return 'CANopen Node ID.'
```

Reply:
This still says "CAN Node ID". CAN does not have node IDs; CANopen does.

Reply:
Oops, fixed now!